docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -76,9 +78,19 @@ \section{Critical Assumptions}
 		\centering
 		\caption{Failure Mode and Effect Analysis Table}
 		\begin{adjustbox}{width=\paperwidth}
-			\begin{tabular}{c|c|c|c|c|c|c}
+			\begin{tabular}{|l|l|l|l|l|l|l|}
 				\hline
-				\textbf{Component} & \textbf{Failure Modes} & \textbf{Effects of Failure} & \textbf{Causes of Failure} & \textbf{Recommended Action} & \textbf{SR} & \textbf{Ref.} \\
+				\textbf{Component}              & \textbf{Failure Modes}                                   & \textbf{Effects of Failure}                                                     & \textbf{Causes of Failure}                                                                   & \textbf{Recommended Action}                                                             & \textbf{SR} & \textbf{Ref.} \\
+				\hline
+				General                         & Loss or theft of Personal Identifiable Information (PII) & Legal consequences, loss of user trust                                          & Malicious internal or external actors                                                        & Encryption of PII data at rest and in transit                                           & ~           & ~             \\
```

Comment:
I think `Malicious internal or external actors` is too vague of a `Cause of Failure`.

Reply:
Updated to be more specific